ambry-router/src/main/java/com/github/ambry/router/NonBlockingRouterMetrics.java

```diff
@@ -312,6 +325,7 @@ public NonBlockingRouterMetrics(ClusterMap clusterMap, RouterConfig routerConfig
     getEncryptedBlobWithSegmentErrorCount =
         metricRegistry.counter(MetricRegistry.name(GetBlobOperation.class, "GetEncryptedBlobWithSegmentErrorCount"));
     deleteBlobErrorCount = metricRegistry.counter(MetricRegistry.name(DeleteOperation.class, "DeleteBlobErrorCount"));
+    undeleteBlobErrorCount = metricRegistry.counter(MetricRegistry.name(DeleteOperation.class, "UndeleteBlobErrorCount"));
```

Comment:
make sure all recently added metrics in this file have been updated to either `UndeleteOperation.class` or `UndeleteManager.class`